landing-strip/core/src/main/java/com/novoda/landingstrip/DynamicTabsContainer.java

```diff
@@ -0,0 +1,24 @@
+package com.novoda.landingstrip;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.ViewGroup;
+import android.widget.LinearLayout;
+
+public class DynamicTabsContainer extends TabsContainer {
```

Comment:
is this the regular (old, standard) container, i.e. that will scroll and is the opposite of the new FixedWidthTabContainer? If so, how about ScrollingTabsContainer to match ScrollingLandingStrip? (as FixedWidthLandingStrip uses FixedWidthTabContainer)